packages/inter-protocol/src/proposals/addAssetToVault.js

```diff
@@ -262,7 +262,11 @@ export const addAssetToVault = async (
   const stable = await stableP;
   const vaultFactoryCreator = E.get(vaultFactoryKit).creatorFacet;
   await E(vaultFactoryCreator).addVaultType(interchainIssuer, oracleBrand, {
-    debtLimit: AmountMath.make(stable, 0n),
+    // DONOTMERGE used for testing, but this script will run in production.
+    // Must the debtLimit start at zero?
```

Comment:
In the PSM release, we decided it was cost-effective to have a >0 default debtLimit, provided it represents a manageable economic risk. We used 1000IST there.

https://github.com/Agoric/agoric-sdk/blob/bdf9cc9c914227990bd6af0030923290d11603d8/packages/inter-protocol/src/proposals/startPSM.js#L59

Even 250IST or 50IST might suffice for some vaults testing.

The `addAssetToVault` function should have a `config` argument that can easily be used by a proposer to the BLDer DAO to choose the initial debt limit.

Reply:
SGTM, I'll make it 1000IST like startPSM. 

I agree it should be a config option. I wrestled a bit with introducing it so I propose it be created when needed.